packages/features/landing_page_screen/lib/src/components/main_overflow_menu.dart

```diff
@@ -45,7 +47,18 @@ class _MainOverFlowMenuState extends ConsumerState<MainOverflowMenu> {
     String version = packageInfo.version;
     switch (option) {
       case MainOverflowMenuOption.rate:
-        LaunchReview.launch(androidAppId: androidAppId, iOSAppId: iOSAppId);
+        if (Platform.isAndroid || Platform.isIOS) {
+          final appId = Platform.isAndroid ? androidAppId : iOSAppId;
+          final url = Uri.parse(
+            Platform.isAndroid
+                ? 'market://details?id=$appId'
+                : 'https://apps.apple.com/app/id$appId',
```

Comment:
I know the id was already there before your PR, but could you please change it so it directs to:
https://apps.apple.com/at/app/pocket-code/id1117935892
We don't have the Pocket Paint app in the app store yet, so for testing purposes let's use Pocket Code's id